client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -11,8 +11,8 @@ import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import schema from './schema.json';
 
 const noop = () => {};
+const jitmSchema = schema;
```

Comment:
another code style nit: just use the `schema` variable directly, no need to assign it to another variable.